pkg/rpc/response/errors.go

```diff
@@ -38,19 +42,18 @@ var (
 	ErrUnknown = NewSubmitError(-500, "Unknown error.")
 )
```

Comment:
We can add more error variables now for clients to use with `errors.Is()`. Then the list will be corrected/expanded.

Reply:
I've extended `(e *Error) Is(target error) bool`, it is done in a separate commit, is it OK?

Reply:
It should only work with `response.Error`, server-side things are irrelevant for this code.